.claude/skills/claude-hooks/SKILL.md

```
@@ -123,78 +124,128 @@ scripts/
+      {
+        "hooks": [{ "type": "command", "command": ".claude/hooks/stop.sh" }]
+      }
+    ]
   }
 }
 ```
 
+### Hook Input/Output Formats
+
+**PostToolUse (track-changes)**:
+```json
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)